server/plugin/command.go

```diff
@@ -310,22 +330,54 @@ func (p *Plugin) handleSubscribesAdd(_ *plugin.Context, args *model.CommandArgs,
 
 	ctx := context.Background()
 	githubClient := p.githubConnectUser(ctx, userInfo)
+	user, appErr := p.API.GetUser(args.UserId)
+	if appErr != nil {
+		return appErr.Error()
+	}
+	owner, repo := parseOwnerAndRepo(parameters[0], baseURL)
+	previousSubscribedEvents, err := p.getSubscribedFeatures(args.ChannelId, owner, repo)
+	if err != nil {
+		return err.Error()
+	}
+
+	var previouslySubscribedEventMessage string
+	if previousSubscribedEvents != "" {
+		previouslySubscribedEventMessage = fmt.Sprintf("\nThe previous subscription with: %s was overwritten.\n", previousSubscribedEvents.FormattedString())
+	}
 
-	owner, repo := parseOwnerAndRepo(parameters[0], config.getBaseURL())
 	if repo == "" {
-		if err := p.SubscribeOrg(ctx, githubClient, args.UserId, owner, args.ChannelId, features, flags); err != nil {
+		if err = p.SubscribeOrg(ctx, githubClient, args.UserId, owner, args.ChannelId, defaultEvents, flags); err != nil {
 			return err.Error()
 		}
+		orgLink := baseURL + owner
+		subscriptionSuccess := fmt.Sprintf("@%v subscribed this channel to [%s](%s) with the following events: %s.", user.Username, owner, orgLink, defaultEvents.FormattedString())
+
+		if previousSubscribedEvents != "" {
+			subscriptionSuccess += previouslySubscribedEventMessage
+		}
+
+		post := &model.Post{
+			ChannelId: args.ChannelId,
+			UserId:    p.BotUserID,
+			Message:   subscriptionSuccess,
+		}
 
-		return fmt.Sprintf("Successfully subscribed to organization %s.", owner)
+		if _, appErr = p.API.CreatePost(post); appErr != nil {
+			p.API.LogWarn("error while creating post", "post", post, "error", appErr.Error())
+			return fmt.Sprintf("%s Though there was an error creating the public post: %s", subscriptionSuccess, appErr.Error())
```

Comment:
remove the "Though there"?

Reply:
@catalintomai as we are only creating public post to user while subscribe , but on error don't want to make a public post it may cause spam messages for other user